config/platform/myparcel.php

```diff
@@ -32,4 +32,11 @@
             CheckoutSettings::PICKUP_LOCATIONS_DEFAULT_VIEW => CheckoutSettings::PICKUP_LOCATIONS_VIEW_LIST,
         ],
     ],
+
+    'sortOrderCarriers' => [
```

Comment:
dit is niet wat ik bedoel, nu zit je alsnog een extra property toe te voegen en werkt dit niet voor de andere platformen. ik bedoel dat je de `allowedCarriers` array in de juiste volgorde zet en zorgt dat dat dus ook de volgorde bepaalt.